pkg/reconciler/apiserversource/resources/receive_adapter_test.go

```diff
@@ -51,14 +51,14 @@ func TestMakeReceiveAdapter(t *testing.T) {
 				{
 					APIVersion: "",
 					Kind:       "Pod",
-					LabelSelector: &metav1.LabelSelector{
+					LabelSelector: metav1.LabelSelector{
```

Comment:
/hold

Is there test coverage of creating a source with no value for labelSelector or controllerSelector? If so, cancel the hold. :)

Reply:
yes there is in the e2e test, which should be fixed now.